audioswitch/src/main/java/com/twilio/audioswitch/bluetooth/BluetoothHeadsetManager.kt

```diff
@@ -302,11 +314,91 @@ internal constructor(
         override fun scoAction() {
             logger.d(TAG, "Attempting to disable bluetooth SCO")
             audioDeviceManager.enableBluetoothSco(false)
-            headsetState = Connected
+            headsetState = HeadsetState.Connected
         }
 
         override fun scoTimeOutAction() {
-            headsetState = AudioActivationError
+            headsetState = HeadsetState.AudioActivationError
         }
     }
 }
+
+internal class BluetoothHeadsetManager_v31plus
```

Comment:
The linter does not like _ in the class names ... perhaps?

```suggestion
internal class BluetoothHeadsetManagerV31Plus
```